src/components/chart/state-history-chart-line.ts

```diff
@@ -114,7 +114,7 @@ export class StateHistoryChartLine extends LitElement {
               },
             },
             min: this.startTime,
-            suggestedMax: this.endTime,
+            max: this.endTime,
```

Comment:
Why can we no longer use `suggestedMax`? I think there was a reason we use this instead of `max` 🤔

Reply:
energy charts already use `max`. I got the idea from them because I couldn't reproduce the bug there while on more-info it was easy